<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransactionDetailsWidget</class>
 <widget class="QWidget" name="TransactionDetailsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QFrame" name="gridFrame">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="horizontalFrame1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="headerPanel" stdset="0">
           <bool>true</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Transaction Details</string>
             </property>
             <property name="headerLabel" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="gridFrame12">
          <property name="darkFrame" stdset="0">
           <bool>true</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_2" columnstretch="2,0">
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item row="12" column="1">
            <widget class="CustomLabel" name="tranSize">
             <property name="text">
              <string/>
             </property>
             <property name="whiteText" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="CustomLabel" name="tranNumOutputs">
             <property name="text">
              <string/>
             </property>
             <property name="whiteText" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="CustomLabel" name="tranFeePerByte">
             <property name="text">
              <string/>
             </property>
             <property name="whiteText" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_15">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Output Amount</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="CustomLabel" name="tranID">
             <property name="text">
              <string/>
             </property>
             <property name="whiteText" stdset="0">
              <bool>true</bool>
             </property>
             <property name="copyToClipboard" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_16">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Input Amount</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Transaction ID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="CustomLabel" name="labelTxPopup">
               <property name="maximumSize">
                <size>
                 <width>13</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="showToolTipQuickly" stdset="0">
                <bool>true</bool>
               </property>
               <property name="quickToolTip" stdset="0">
                <string extracomment="The Transaction ID (TXID) is in big endian notation. It will differ from the user input if the input is little endian."/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_14">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Fees</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_13">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Inputs #</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="CustomLabel" name="tranNumInputs">
             <property name="text">
              <string/>
             </property>
             <property name="whiteText" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="CustomLabel" name="tranOutput">
             <property name="text">
              <string/>
             </property>
             <property name="whiteText" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="label_7">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Size (Virtual)</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="CustomLabel" name="tranFees">
             <property name="text">
              <string/>
             </property>
             <property name="whiteText" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="CustomLabel" name="tranInput">
             <property name="text">
              <string/>
             </property>
             <property name="whiteText" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="label_10">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Fee per byte (s/b)</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_12">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Outputs #</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="labelNbConf">
             <property name="text">
              <string>Confirmations #</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="nbConf">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="horizontalFrame2">
       <property name="darkFrame" stdset="0">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QFrame" name="horizontalFrame3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="headerPanel" stdset="0">
             <bool>true</bool>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Input</string>
               </property>
               <property name="headerLabel" stdset="0">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="CustomTreeWidget" name="treeInput">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <column>
             <property name="text">
              <string>Address</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Amount</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Wallet</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="horizontalFrame">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="headerPanel" stdset="0">
             <bool>true</bool>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Output</string>
               </property>
               <property name="headerLabel" stdset="0">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="CustomTreeWidget" name="treeOutput">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <column>
             <property name="text">
              <string>Address</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Amount</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Wallet</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header>CustomLabel.h</header>
  </customwidget>
  <customwidget>
   <class>CustomTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>CustomTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
